projects/backend/src/main/main.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file? Is it needed by our application or did You accidentally added something needed by ```Intellij```?